cli/command/formatter/service.go

```diff
@@ -92,10 +92,10 @@ ContainerSpec:
 Mounts:
 {{- end }}
 {{- range $mount := .ContainerMounts }}
-  Target = {{ $mount.Target }}
-   Source = {{ $mount.Source }}
-   ReadOnly = {{ $mount.ReadOnly }}
-   Type = {{ $mount.Type }}
+ Target:	{{ $mount.Target }}
+  Source:	{{ $mount.Source }}
+  ReadOnly: {{ $mount.ReadOnly }}
```

Comment:
Looks like this one has to be a tab as well;

```
Mounts:
 Target:	/bar
  Source:	foo
  ReadOnly: false
  Type:		volume
```

Reply:
identation is acting weird since the last change I just pushed works well for me you can see the attached image

![2018-05-12_00_03_30](https://user-images.githubusercontent.com/5016807/39948652-59a26736-5577-11e8-9d89-c221bd61c0b1.jpeg)

Reply:
Yeah, we should probably use a tabwriter for this, so that alignment always works independent of local settings of the shell